### PR TITLE
No more fixtures, do `manage.py seed` instead

### PR DESCRIPTION
**Pull request by noahsark769**
A more intuitive way to seed the database!

You can run `(v)manage.py seed` to load in seed data, or clear the loaded in data with `manage.py seed --clear`. Also you can load seeds specifically:

```
python vmanage.py seed revolvuserprofile
python vmanage.py seed project
python vmanage.py seed payment
```

@vivekraghuram or @andylouisqin or @erictshen can you review this?

### CODE REVIEW

File: revolv/base/management/commands/seed.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+                user.delete()
+            except User.DoesNotExist as e:
+                print "[Seed:Warning] Error in %s when trying to clear: %s" % (self.__class__.__name__, str(e))
+
+
+class ProjectSeedSpec(SeedSpec):
+    """
+    Database seed specification for revolv.project.models.Project
+
+    Creates 4 projects with various settings: Community Dairy, Community Dance Studio,
```
Comment (vivekraghuram) [inline revolv/base/management/commands/seed.py]: I believe that should be "comoonity dairy"

**Comment by vivekraghuram** [comment]
lgtm

File: revolv/base/management/commands/seed.py (hunk 1 above)
Comment (noahsark769) [inline revolv/base/management/commands/seed.py]: lol oops how could I forget

**Comment by noahsark769** [comment]
Yay thanks @vivekraghuram

**Pull request merged**